ansible/roles/ocp4-workload-idm/tasks/workload.yml

```diff
@@ -41,33 +41,42 @@
 
 - name: Install CAs
   shell: |
-    echo {{ idm_admin_password }} | kinit admin
-    ipa-cacert-manage -p {{ idm_dm_password }} install /tmp/DSTRootCAX3.pem -n DSTRootCAX3 -t C,,
-    ipa-cacert-manage -p {{ idm_dm_password }} install /tmp/LEAuthX3.pem -n LEAuthX3 -t C,,
+    echo '{{ idm_admin_password }}' | kinit admin
+    ipa-cacert-manage -p '{{ idm_dm_password }}' install /tmp/DSTRootCAX3.pem -n DSTRootCAX3 -t C,,
+    ipa-cacert-manage -p '{{ idm_dm_password }}' install /tmp/LEAuthX3.pem -n LEAuthX3 -t C,,
     ipa-certupdate -v
   become: True
 
 - name: Install IPA Certificate
   shell: |
-    ipa-server-certinstall -w -d /home/{{ ansible_user }}/idm/certificates/privkey.pem /home/{{ ansible_user }}/idm/certificates/fullchain.pem -p {{ idm_dm_password }} --pin=''
+    ipa-server-certinstall -w -d /home/{{ ansible_user }}/idm/certificates/privkey.pem /home/{{ ansible_user }}/idm/certificates/fullchain.pem -p '{{ idm_dm_password }}' --pin=''
     ipactl restart
   become: True
 
 - name: Install redeploy hook scripts
   template:
     src: ./files/deploy_certs.sh
-    dest: "/home/{{ ansible_user }}/idm/certbot/renewal-hooks/deploy/deploy_certs.sh"
+    dest: "/home/{{ ansible_user }}/idm/certbot/config/renewal-hooks/deploy/deploy_certs.sh"
     mode: 0775
     owner: "{{ ansible_user }}"
 - name: Install redeploy hook ansible components
   copy:
     src: "./files/{{ item }}"
-    dest: "/home/{{ ansible_user }}/idm/certbot/renewal-hooks/deploy/{{ item }}"
+    dest: "/home/{{ ansible_user }}/idm/certbot/config/renewal-hooks/deploy/{{ item }}"
     mode: 0664
     owner: "{{ ansible_user }}"
   loop:
   - deploy_certs.yml
 
+- name: Install AWS python prerequisites
```

Comment:
Should this be in pre_workload.yml?

Reply:
Which version of pip does this use?

Reply:
I'm concerned that we might be trying to re-engineer here...if we refactor at this point we introduce risk.  This uses python2 pip downstream, I'm not sure what the upstream has introduced...they seem to be putting the path to pip to get pip3 IIRC.